### DISCUSSION TITLE
Blowing cache causes desync

### DISCUSSION THREAD

**Discussion by AshTrayBY** [Vdauphin/HeartsAndMinds#1576] [Question and answer] [answered]
Sometimes blowing cache causes desync for all players on server. Seems like server stops any simulation. All AI freezes. Players can move but can't see anothers players actual position - another players seems frozen where they were on the moment of cache explosion. Any ideas why it can happend?

**Comment by Vdauphin** [comment]
Hello,

> Any ideas why it can happend?

No, never saw that
May be a broken mod

Cheer

**Comment by Playmakinggg** [comment]
Can send, have saw this recently; not sure if it's a mod or a game-mode specific issue.

**Reply by Vdauphin** [reply, marked as answer]
Hello,
This has been fix in last released 1.23.7 https://github.com/Vdauphin/HeartsAndMinds/pull/1588
See https://github.com/Vdauphin/HeartsAndMinds/discussions/1587
Cheers

**Reply by CLAFMB** [reply]
Hello, this problem still exists after the att, because mine freezes after 2 to 3 hours of play, everything freezes except my character, the rest everything stays static or moving in the same place

**Reply by Vdauphin** [reply]
Hello,

Can you create an issue or investigate on your side to find the root cause of this behaviour?

Cheers

**Reply by Playmakinggg** [reply]
Hey, still having this issue as of 2024/07/24. Would you prefer I make a issue with my console logs or how would you like me to go about getting this info to you? 

Appreciate you constantly working on this project and continuing to make one of, if not the best gamemode available.

**Reply by CLAFMB** [reply]
Good evening, I investigated the cause and found a mod with a bug that when an entity that used it spawned, the game froze.

**Reply by Vdauphin** [reply]
> Would you prefer I make a issue with my console logs or how would you like me to go about getting this info to you?

You can create an issue, but you must fill the issue template overwise, it is pointless
